comictaggerlib/ui/taggerwindow.ui

```diff
@@ -52,37 +52,23 @@
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
-           <layout class="QFormLayout" name="formLayout">
+           <layout class="QFormLayout" name="source_style_formLayout">
```

Comment:
The comment on taggerwindow lines +230 to +241 removes the need for this to be named but.
`style_layout` is a much more manageable name